### DISCUSSION TITLE
Time Tracking limited to one Task

### DISCUSSION THREAD

**Discussion by mjut** [kanboard/kanboard#5556] [Q&A]
I am looking into Kanboard for a possible project management app for our team of 10 employees.
What I am finding confusing is the time tracking feature. How can I use this properly? I am running into the issue that when starting the tracking I am forgetting about it. I then browse to a different project and start another time tracker.
How can I avoid this? It shouldn't be possible to track multiple task at simultaneously – additionally, I should actually see the time being tracked on every screen. something like a „status“ in the top-bar of Kanboard?
It would be cool to hear others opinions on this. Or How do you track times accurately?

Thanks!

**Comment by jameswar54502** [comment]
I find the idea of limiting **[time tracking software](https://desktrack.timentask.com/time-tracking-software)** to one task quite interesting! While it might seem restrictive at first, focusing on a single task can actually enhance productivity by minimizing distractions and fostering deeper concentration. It allows individuals to fully immerse themselves in their work, leading to higher-quality outcomes.
